src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -38,24 +39,30 @@
 FILENAME_LENGTH = 255
 LOG_NAME_REPLACE_STR = "##NAME##"
 DEFAULT_PROCESS_TIMEOUT = 3600
+DEFAULT_SUITE_TIMEOUT = 3600
 DEFAULT_TEST_TIMEOUT = 900
 MAX_LENGHT = 4096 if not constants.IS_WIN else 8191
 
 def parse_arguments():
     parser = ArgumentParser()
     exec_file_path_help = "Path to the test executable file"
     cache_path_help = "Path to the cache file with test_name list sorted by execution time. .lst file!"
-    worker_num_help = "Worker number. Default value is `cpu_count-1` "
+    worker_num_help = "Worker number. Default value is `cpu_count` "
```

Comment:
Can leads to hung of other os processes

Reply:
I think it depends in OS, usually it should not happen, and we would need to reuse all the cores on the different runners and it means that changing of the runner requires changing command line arguments.